dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -588,6 +588,22 @@ def cache_cards(cfg):
             "key": issue.fields.assignee.key,
             "name": issue.fields.assignee.name
         }
+
+        # Get contributors
+        if issue.fields.customfield_12315950:
+            for engineer in issue.fields.customfield_12315950:
+                contributor = {
+                    "displayName": engineer.displayName,
+                    "key": engineer.key,
+                    "name": engineer.name
+                }
+        else:
+            contributor = {
+                "displayName": "None",
```

Comment:
so I know this habit leaked in through a few spots, but I'm not a big fan. Any reason we can't use the None type, rather than make it a string? Also makes the jinja2 line shorter, since its treated like a boolean and won't need to do a string comparison on it.

Reply:
Thanks for bringing this up, initially I thought the None type wouldn't work with our tables, but it looks like it's working. I'll fix this and add an issue to fix it elsewhere in the code